src/WebJobs.Script/Rpc/MessageExtensions/RpcMessageConversionExtensions.cs

```diff
@@ -159,5 +160,21 @@ public static TypedData ToRpc(this object value)
             }
             return typedData;
         }
+
+        public static BindingInfo ToBindingInfo(this BindingMetadata bindingMetadata)
+        {
+            BindingInfo bindingInfo = new BindingInfo
+            {
+                Direction = (BindingInfo.Types.Direction)bindingMetadata.Direction,
+                Type = bindingMetadata.Type
+            };
+
+            if (bindingMetadata.DataType != null)
+            {
+                bindingInfo.DataType = (BindingInfo.Types.DataType)bindingMetadata.DataType;
```

Comment:
Update/Add Node E2E test that sets Datatype. This verification makes sure adding this property does not break existing language workers.